src/app/(post-login)/submission/components/ProgramMenu.tsx

```diff
@@ -259,18 +269,18 @@ const MenuContent = ({
 								{/* {clinicalDataHasErrors && <Icon name="exclamation" fill="error" width="15px" />} */}
 							</StatusMenuItem>
 						}
-						selected={PROGRAM_CLINICAL_DATA_PATH === pathname}
+						selected={pathnameLastSegment === 'clinical-data'}
 					/>
 				</Link>
 			)}
 
 			{/** Manage Program */}
 			{userCanManageProgram && (
-				<Link href={PROGRAM_MANAGE_PATH.replace(PROGRAM_SHORT_NAME_PATH, programName)}>
+				<Link href={getProgramPath(PROGRAM_MANAGE_PATH)}>
 					<MenuItem
 						level={3}
 						content="Manage Program"
-						selected={PROGRAM_MANAGE_PATH === pathname}
+						selected={pathnameLastSegment === 'manage'}
 					/>
 				</Link>
 			)}
```

Comment:
This is fine. your original `replace` in the `href` prop was valid. 

It was just the `selected` prop part that needed to have the dynamic part injected which is now not needed because it's been reverted to how it was before.